src/Cake.Common/Tools/NUnit/NUnit3ProcessOption.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace Cake.Common.Tools.NUnit
+{
+    /// <summary>
+    /// Represents the various ways NUnit loads tests in processes.
+    /// </summary>
+    public enum NUnit3ProcessOption
+    {
+        /// <summary>
+        /// A separate process is created for each test assembly. This is the default.
+        /// </summary>
+        Multiple,
```

Comment:
Would prefer this enum value to explicitly be assigned the value of `0` to avoid problems if someone reorder this in the future.